CommandHelper/LocalPackages/commands/buffs/doctormanhattan.ms

```diff
@@ -10,5 +10,17 @@
             'op': 'true',
             'speech': "The existence of life is a highly overrated phenonenon.",
             'powers': array(),
+                "/kill",
+                "/gmc",
+                "/gms",
+                "/tp",
+                "/fly",
+                "/flyspeed <number>"
+            ),
             'buffs': array()
-        )
+                "/effect ". @plr ." 5 9999 25 true",
+                "/effect ". @plr ." 10 9999 4 true",
+                "/effect ". @plr ." 11 9999 5 true",
```

Comment:
this should be set no higher than 4. 4 = invincible. 5 = invincible. 6 = invincible, etc